items.rb

```diff
@@ -0,0 +1,22 @@
+require 'date'
+
+class ClassName
+  def initialize(id, date, archived: true)
+    @id = id
+    @publish_date = date
```

Comment:
I'm so sorry for not pointing out this earlier 😷 The date is not printing out correctly. As of now, the result that we are getting is something like mentioned below 🔽

![image](https://user-images.githubusercontent.com/106119105/211408824-a19b544a-c2eb-4eac-b63d-7a33ec502b61.png)

 This can easily be fixed by `Date.parse` method as mentioned [here](https://www.rubyguides.com/2015/12/ruby-time/)
 Instead of `@publish_date = date`, do this `@publish_date = Date.parse(date)` and we can easily get the following result 🔽

 
![date](https://user-images.githubusercontent.com/106119105/211409209-143572d1-af60-46fe-900c-22d319064da1.PNG)